src/components/Alerts/UserActionAlert.js

```diff
@@ -26,6 +27,9 @@ const UserActionAlert = ({
     case 'delete_expense':
       // required props: errorOccurred, closeCallback, expense
       return <ExpenseDeletedAlert {...baseProps} {...expense} />;
+    case 'edit_expense':
+      // required props: errorOccurred, closeCallback
+      return <ExpenseEditedAlert {...baseProps} />;
```

Comment:
just from these few lines, I am wondering why we need to pass `expense` to "delete" but not to "edit". i know that it is passed as part of `baseProps`, but why do these two signatures differ?

Reply:
Maybe the prop name isn't descriptive enough. Basically, we are passing expense information to `ExpenseDeletedAlert` because we want to display the name, date and amount of the deleted expense as a form of confirmation to the user. The reason why we don't pass `expense` to the other alerts is because (as of this very moment) we haven't seen a need to provide the user with such detailed feedback when they add or edit an expense.

If `ExpenseDeletedAlert` is the only component that requires expense information, we could just rename the prop: `deletedExpense`.